src/test/java/rx/internal/util/IndexedRingBufferTest.java

```diff
@@ -191,11 +191,7 @@ public Boolean call(String t1) {
             @Override
             public Boolean call(String t1) {
                 list.add(t1);
-                if (i++ == 2) {
-                    return false;
-                } else {
-                    return true;
-                }
+                return i++ != 2;
```

Comment:
Well I didn't want to change any existing logic there... 
I don't know code base well enough to feel comfortable enough.
I could take that "i++" one separate line higher.